### PR TITLE
feat(settings): Update account preferences help link, add Glean click event

### PR DESCRIPTION
**Pull request by LZoog**
Because:
* Accounts has its own dedicated help link we should use instead
* We want a metrics event for when users click on this link

This commit:
* Updates the link and adds the Glean event

closes FXA-10220, closes FXA-10221

### CODE REVIEW

File: packages/functional-tests/pages/settings/layout.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ export abstract class SettingsLayout extends BaseLayout {
   }
 
   get helpLink() {
-    return this.page.getByTestId('header-sumo-link');
+    return this.page.getByRole('link', { name: 'Help' });
```
Comment (vpomerleau) [inline packages/functional-tests/pages/settings/layout.ts:14]: Nice 💎

File: packages/fxa-settings/src/components/Settings/HeaderLockup/index.test.tsx [hunk 2]
```diff
@@ -34,9 +34,21 @@ describe('HeaderLockup', () => {
     expect(headerMenu).toBeInTheDocument();
     expect(headerMenu).toHaveAttribute('title', 'Site navigation menu');
 
+    expect(screen.getByRole('link', { name: 'Help' })).toHaveAttribute(
+      'href',
+      'https://support.mozilla.org/products/mozilla-account'
```
Comment (vpomerleau) [inline packages/fxa-settings/src/components/Settings/HeaderLockup/index.test.tsx:39]: Do we want/need any UTM params for SUMO?

**Review by vpomerleau — APPROVED** [review]
LGTM!

File: packages/fxa-settings/src/components/Settings/HeaderLockup/index.test.tsx (hunk 2 above)
Comment (LZoog) [reply packages/fxa-settings/src/components/Settings/HeaderLockup/index.test.tsx:39]: There weren't any listed in the ticket, so I don't think so. We'll get metrics info on our side from the new Glean event added here 🤷‍♀️

**Pull request merged**